packages/react-native-sdk/src/components/Call/CallControls/ToggleCameraFaceButton.tsx

```diff
@@ -47,7 +47,9 @@ export const ToggleCameraFaceButton = ({
     <Restricted requiredGrants={[OwnCapability.SEND_VIDEO]}>
       <CallControlsButton
         onPress={onPress}
-        color={direction === 'back' ? colors.overlay_dark : colors.static_white}
+        color={
+          direction === 'back' ? colors.overlay_medium : colors.static_white
```

Comment:
we should avoid any strong color reference here like 

```
colors.static_white
```

rather it should be generic like `colors.cardBackground` etc